ontology/yaml/resources/METERS/entity_types/WM.yaml

```diff
@@ -20,6 +20,9 @@ WM_STANDARD:
   is_canonical: true
   uses:
   - water_volume_accumulator
+  opt_uses:
+  - failed_alarm
+  - supply_water_flowrate_sensor
```

Comment:
does it need to be supply or can this just be `flowrate_sensor`?

Reply:
Yes it can be .
Thank you .
Updated the changes.